### ISSUE TITLE
Identifying patient centric resources 

### ISSUE THREAD

**Issue by zavoraad** [databrickslabs/dbignite#21]
Thinking "entry.request.url" should be "entry.resource.resourceType" 

https://github.com/databrickslabs/dbignite/blob/c0c6541501e435fe490fae1abb717df950241a78/dbignite/utils.py#L31

Supporting FHIR info - https://www.hl7.org/fhir/request.html#12.3.1 "This is NOT a resource. It is not part of the FHIR schema and cannot appear directly in FHIR instances."

**Comment by zavoraad** [comment]
```
Main class:
org.apache.spark.api.python.PythonGatewayServer
Arguments:

Spark config:
(spark.app.name,PySparkShell)
(spark.app.submitTime,1679948647848)
(spark.driver.bindAddress,127.0.0.1)
(spark.files,file:///Users/aaron.zavora/.ivy2/jars/io.delta_delta-core_2.12-2.2.0.jar,file:///Users/aaron.zavora/.ivy2/jars/io.delta_delta-storage-2.2.0.jar,file:///Users/aaron.zavora/.ivy2/jars/org.antlr_antlr4-runtime-4.8.jar)
(spark.jars,file:///Users/aaron.zavora/.ivy2/jars/io.delta_delta-core_2.12-2.2.0.jar,file:///Users/aaron.zavora/.ivy2/jars/io.delta_delta-storage-2.2.0.jar,file:///Users/aaron.zavora/.ivy2/jars/org.antlr_antlr4-runtime-4.8.jar)
(spark.master,local[*])
(spark.repl.local.jars,file:///Users/aaron.zavora/.ivy2/jars/io.delta_delta-core_2.12-2.2.0.jar,file:///Users/aaron.zavora/.ivy2/jars/io.delta_delta-storage-2.2.0.jar,file:///Users/aaron.zavora/.ivy2/jars/org.antlr_antlr4-runtime-4.8.jar)
(spark.submit.deployMode,client)
(spark.submit.pyFiles,/Users/aaron.zavora/.ivy2/jars/io.delta_delta-core_2.12-2.2.0.jar,/Users/aaron.zavora/.ivy2/jars/io.delta_delta-storage-2.2.0.jar,/Users/aaron.zavora/.ivy2/jars/org.antlr_antlr4-runtime-4.8.jar)
(spark.ui.showConsoleProgress,true)
Classpath elements:
file:///Users/aaron.zavora/.ivy2/jars/io.delta_delta-core_2.12-2.2.0.jar
file:///Users/aaron.zavora/.ivy2/jars/io.delta_delta-storage-2.2.0.jar
file:///Users/aaron.zavora/.ivy2/jars/org.antlr_antlr4-runtime-4.8.jar


Setting default log level to "WARN".
To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
23/03/27 16:24:09 ERROR Utils: Aborting task
java.io.IOException: Failed to connect to /198.105.254.26:59638
	at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:288)
	at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:218)
	at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:230)
```

**Comment by zavoraad** [comment]
Fixed in version 0.2.0
